client/core/core.go

```diff
@@ -3132,20 +3154,24 @@ func (c *Core) discoverAccount(dc *dexConnection, crypter encrypt.Crypter) (bool
 			}
 			return false, newError(authErr, "unexpected authDEX error: %w", err)
 		}
-		if dc.acct.isSuspended {
-			c.log.Infof("HD account key for %s was reported as suspended. Deriving another account key.", dc.acct.host)
+		// do not skip key if tier is 0 and bonds will be used
+		if dc.acct.tier < 0 || (dc.acct.tier < 1 && dc.apiVersion() < serverdex.BondAPIVersion) {
+			c.log.Infof("HD account key for %s was has tier %d (not able to trade). Deriving another account key.",
+				dc.acct.host, dc.acct.tier)
```

Comment:
> I guess a client could go far into bad behavior territory now, and have a very negative score. They would need more bond to overcome this. Maybe they would rather make a new account than try to keep up with the higher "fees" they have incurred.
> 
> Conversely, maybe the user would like to lock more bonds rather than skip this account.

Correct that there's actually a choice here because no account is permanently suspended anymore... no more "ban" concept.  In this PR, I'm trying to preserve current behavior of the client, just using the new messages and types.  In this case I'm considering negative tier to mean suspended permanently (not worth posting more bond to get out of the hole).  In following bond PRs we'll certainly get into this more.

> Can tier 0 mean that either the account has no bonds or has enough negative credit that it is shifted to zero?

Yes, can mean either.  Tier is just a balance of active bond strength with penalizations.

> Does a tier less than zero mean definite bad behavior?

That's the only way for the formula to go negative, yes.

https://github.com/decred/dcrdex/blob/72603a9c7e6f668ef8dbb75ceb1f51b3d910d6d1/server/auth/auth.go#L846-L853